application-task-default/src/main/java/com/xwiki/task/internal/notifications/TaskChangedEventDescriptor.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+
+package com.xwiki.task.internal.notifications;
+
+import javax.inject.Named;
+import javax.inject.Singleton;
+
+import org.xwiki.component.annotation.Component;
+import org.xwiki.eventstream.RecordableEventDescriptor;
+
+/**
+ * Descriptor for the event {@link TaskChangedEvent}.
+ *
+ * @version $Id$
+ * @since 3.5.2
+ */
+@Component
+@Singleton
+@Named("com.xwiki.task.internal.notifications.TaskChangedEventDescriptor")
+public class TaskChangedEventDescriptor implements RecordableEventDescriptor
+{
+
+    @Override
+    public String getEventType()
+    {
+        return TaskChangedEvent.class.getCanonicalName();
```

Comment:
The notification filter doesn't actually do anything, turning it on or off will still send the `TaskChangedEvent` notification.